static/css/devhub/new-landing/navigation.less

```diff
@@ -81,9 +162,38 @@
     }
   }
 
-  &.scheme-medium {
-    ul li a {
-      color: @color-scheme-medium-text;
+  @media @notlarge {
+  &:target {
+    height: 100vh;
+    margin-bottom: 24px;
+
+    .in-hamburger-menu {
+        display: list-item;
     }
-  }
+
+    .Logo span {
+      display: none;
```

Comment:
I think we should not have this rule. It is confusing when we open the menu and the title disappears, yet we can click on it and it reloads the page. I suggest we simply keep it:

![Screen Shot 2019-10-25 at 12 25 19](https://user-images.githubusercontent.com/217628/67564360-0f0df600-f723-11e9-8b45-7133c05dbe58.png)

Reply:
that's how it is on the mocks so you're have to talk to UX about that 😛

Reply:
cc @aaronrbenson

Reply:
@willdurand hey could you point me to the mocks for this so I can get a clearer picture of the intent before making a suggestion?

Reply:
Okay, I see now! Yeah I tend to agree .. having the title disappear doesn't seem to be necessary. Maybe it was to avoid running out of space and the title colliding with the "x"? If that doesn't seem to be an issue and the "x" always ends up with enough white-space around it to be easily tapped, we can keep the title in there for the menu.

Reply:
okay, always shown now https://github.com/mozilla/addons-server/pull/12703/commits/9984edfb6aea12ff52a9133d1638ee13ce41c921